lib/compat.js

```diff
@@ -0,0 +1,136 @@
+const through = require('through2');
+const ls = require('list-stream');
+
+/**
+ * Datastar API compatibility
+ * @class
+ */
+class Compat {
```

Comment:
You don't need this anymore, right?

Reply:
@msluther oops, forgot to remove this file. it isn't used anymore